onnxruntime/core/framework/allocatormgr.cc

```diff
@@ -66,6 +67,9 @@ AllocatorPtr CreateAllocator(const AllocatorCreationInfo& info) {
                                              initial_chunk_size_bytes,
                                              max_dead_bytes_per_chunk,
                                              initial_growth_chunk_size_bytes));
+#else
+      ORT_THROW("StreamAwareArena should be transparent to minimal build.");
```

Comment:
But minimal build typical doesn't have exceptions. So, here it will simply crash?

Reply:
A minimal build can optionally disable exceptions (but typically if you're wanting the minimum size you'd do that). 

The ORT_THROW macro in that build will log the error and abort, which is all it can do.

Reply:
Thanks for clarification. Then I just close this comment